### PR TITLE
Help command

### PR DESCRIPTION
**Pull request by wnchan**

### CODE REVIEW

File: src/test/java/seedu/address/ui/HelpWindowTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package seedu.address.ui;
```
Comment (neyapraveen) [inline src/test/java/seedu/address/ui/HelpWindowTest.java:1]: Test cases should help with coverage. Nice!

**Review by neyapraveen — COMMENTED** [review]
Overall LGTM! Just waiting for the CI to finish to confirm.

File: docs/images/helpMessage.png
Comment (Bearypop) [inline docs/images/helpMessage.png:1]: The words look a bit small here. Maybe can increase the font size? If it's actually bigger in the app, then never mind.

**Review by Bearypop — COMMENTED** [review]
Nice!

**Pull request merged**